eurec4a_environment/variables/boundary_layer/inversion_height.py

```diff
@@ -0,0 +1,25 @@
+def find_inversion_height_grad_RH(
+    ds, altitude="alt", rh="rh", smooth=False, z_min=1500, z_max=4000.0
+):
+
+    """
+        Returns inversion height defined as the maximum in the vertical gradient of RH
+        """
+    ds_lowertroposhere = ds.sel({altitude: slice(z_min, z_max)})
+    window = 10  # window for convolution
```

Comment:
I would make `window` a kwarg for the function. Maybe you could remove `smooth` and have `smoothing_win_size=None` and then only apply smoothing if the user passes in a number? What do you think?

Reply:
Nice idea yes!